.yeoman/templates/utilities/src/index.test.js

```diff
@@ -0,0 +1,7 @@
+import <%= packageName %> from './index';
+
+describe('<%= packageName %>', () => {
+  it('should work', () => {
+    expect(<%= packageName %>()).toEqual(null);
```

Comment:
Calling a function without a return value, return `undefined` instead of `null`.